### PR TITLE
rfsm: add upper bound on lascar

### PR DESCRIPTION
**Pull request by mseri**
see https://github.com/ocaml/opam-repository/pull/14827

Signed-off-by: Marcello Seri <marcello.seri@gmail.com>

### CODE REVIEW

**Comment by kit-ty-kate** [comment]
Could you also do it for `1.6-alpha-3` ?

**Comment by mseri** [comment]
It is already bounded with >= 0.6. That is why I did not touch it

**Comment by kit-ty-kate** [comment]
> It is already bounded with >= 0.6. That is why I did not touch it

But it doesn't work with `0.6.0`. However it works with `0.6-alpha`. See: http://check.ocamllabs.io/log/1567892640-218c496f798f5f1e64e4120e6230952138cef4cf/4.09/good/rfsm.1.6-alpha-3

**Comment by mseri** [comment]
I see. I made the bound strict on 0.6-alpha

File: packages/rfsm/rfsm.1.0/opam [hunk 1]
```diff
@@ -26,7 +26,7 @@ depends: [
   "ocamlbuild" {build}
   "ocamlgraph"
   "menhir"
-  "lascar"
+  "lascar" {< "0.6"}
```
Comment (kit-ty-kate) [inline packages/rfsm/rfsm.1.0/opam]:
```suggestion
  "lascar" {< "0.6.0"}
```

File: packages/rfsm/rfsm.1.4.2/opam [hunk 2]
```diff
@@ -21,7 +21,7 @@ depends: [
   "ocamlbuild" {build}
   "ocamlgraph"
   "menhir"
-  "lascar"
+  "lascar" {< "0.6"}
```
Comment (kit-ty-kate) [inline packages/rfsm/rfsm.1.4.2/opam]:
```suggestion
  "lascar" {< "0.6.0"}
```

File: packages/rfsm/rfsm.1.5/opam [hunk 3]
```diff
@@ -21,7 +21,7 @@ depends: [
   "ocamlbuild" {build}
   "ocamlgraph"
   "menhir"
-  "lascar"
+  "lascar" {< "0.6"}
```
Comment (kit-ty-kate) [inline packages/rfsm/rfsm.1.5/opam]:
```suggestion
  "lascar" {< "0.6.0"}
```

**Comment by kit-ty-kate** [comment]
Thanks a lot!

**Pull request merged**